processing/src/main/java/io/druid/segment/column/SimpleDictionaryEncodedColumn.java

```diff
@@ -177,6 +176,13 @@ public ValueMatcher makeValueMatcher(Predicate<String> predicate)
           return DimensionSelectorUtils.makeValueMatcherGeneric(this, predicate);
         }
 
+        @Override
+        public boolean isNull()
+        {
+          IndexedInts row = getRow();
```

Comment:
Should be `return false`, DimensionSelector is analog of Object in isNull() semantics. isNull() is only relevant for primitive. Might be worth adding this to javadocs.

Reply:
removed isNull, it was not needed to override it here.